reference/src/glossary.md

```diff
@@ -146,6 +146,11 @@ Moreover, such unsafe code must not return a non-UTF-8 string to the "outside" o
 To summarize: *Data must always be valid, but it only must be safe in safe code.*
 For some more information, see [this blog post](https://www.ralfj.de/blog/2018/08/22/two-kinds-of-invariants.html).
 
+#### Soundness (of code / of a library)
+
+We say that a library (can be an individual function) is *sound* if it is impossible for safe code to cause Undefined Behavior using its public API.
```

Comment:
The part in parens feels awkward. How about "can be" changing to "or"

Reply:
Like this?

Reply:
Yes, excellent.